llm/mcloud/mcli-1b.yaml

```diff
@@ -0,0 +1,26 @@
+integrations:
+  - integration_type: git_repo
+    git_repo: mosaicml/examples
+    git_branch: v0.0.2
```

Comment:
does this work for tags? There isn't actually a branch with this name

Reply:
Yup it works!